prow_config.yaml

```diff
@@ -1,15 +1,19 @@
 # This file configures the workflows to trigger in our Prow jobs.
 # see kubeflow/testing/py/run_e2e_workflow.py
 python_paths:
+  # Need to place kubeflow/testing in front of kubeflow/kubeflow so that the
+  # package can be correctly located.
+  - kubeflow/testing/py
   - kubeflow/kubeflow/py
 workflows:
-  # Run unittests
-  # TODO(jlewi): Need to add step to run go and python unittests
-  - app_dir: kubeflow/kubeflow/testing/workflows
-    component: unit_tests
-    name: unittests
-    params:
-      workflowName: unittest
+  # Run unittests for the shared frontend code
+  - py_func: kubeflow.kubeflow.ci.shared_ui_tests.create_workflow
+    name: shared_ui_tests
+    job_types:
+      - presubmit
+    include_dirs:
+      - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
```

Comment:
The `include_dirs` will contain the directory that you wish to kick off the workflow.

But apprently in this PR, you haven't made any change to `components/crud-web-apps/common/frontend/kubeflow-common-lib/*`

Then there's no workflow